packages/edit-widgets/src/components/widget-block-editor-settings/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext } from '@wordpress/element';
```

Comment:
Using a context for the widgets settings does make sense if we forsee us having a complex nested component tree for the widgets page. Is this the case or will be just passing the settings down to the block editors?

Also, I want to remind that we did start with "Context" in the intial versions of the editor settings as well and we ultimately changed to use the data module because we needed to make updates to the settings from the component and we didn't want to reinvent the wheel as this just meant we need global state that can be updated (data store).

As always, I like to start with the simplest possible solution. Isn't a "settings" prop a solution?